examples/player/simid_player.html

```diff
@@ -42,6 +42,24 @@
       simidPlayer.playAd();
     }
 
+    function closeAd() {
```

Comment:
Seems odd to immediately close the ad after init. Maybe these buttons just shouldn't be active until after the ad has been initialized -- then just to be safe you can early return here if simidPlayer doesnt exist

Reply:
fixed